pycytominer/cyto_utils/util.py

```diff
@@ -294,8 +294,17 @@ def extract_image_features(image_feature_categories, image_df, image_cols, strat
     image_features_df = image_df[image_features]
 
     # Add "Image_" prefix to the features
+    # image_features_df.columns = [
+    #     f"Image_{x}" if not x.startswith("Image_") else x
+    #     for x in image_features_df.columns
+    # ]
```

Comment:
I'm in favor of cleaning old commented code. I think keeping things clean incentivizes other contributors to write clean, well-documented, and meaningful code.

Any reason to keep this in other than we might want to revert back at some point?

Reply:
Good catch! I forgot to delete it before committing. https://github.com/cytomining/pycytominer/pull/171/commits/bd73a49482759a4821e16dd9c96cde257a4d8986